test/functional/api/cas/cache.py

```diff
@@ -66,6 +65,23 @@ def get_status(self):
         status = self.get_statistics().config_stats.status.replace(' ', '_').lower()
         return CacheStatus[status]
 
+    @staticmethod
+    def calculate_metadata_max_size(cache_size: Size, cache_line_size: CacheLineSize,
+                                    number_of_cores=4096):
```

Comment:
This is a constant, no point in making this a function parameter. I would rename it to "max_number_of_cores"